ghost/core/content/plugins/review/card.css

```
@@ -0,0 +1,213 @@
+.plugin-card-scope,
+.plugin-card-scope * {
+  box-sizing: border-box;
+}
+
+.review-card {
+  background: #ffffff;
+  border: 2px solid #e2e8f0;
+  border-radius: 12px;
+  padding: 32px;
+  margin: 32px 0;
+  font-family: 'Inter', system-ui, -apple-system, sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)